app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> None:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(players: list) -> None:
+    [player.play_elf_song() for player in players]
```

Comment:
you don't need comprehension, cuz you don't even use list which is created here. So it is just a memory waste.